modular_skyrat/master_files/code/datums/traits/neutral.dm

```diff
@@ -21,6 +21,15 @@
 	mob_trait = TRAIT_IRONASS
 	icon = "hand-paper"
 
+/datum/quirk/personalspace
+	name = "Personal Space"
+	desc = "You'd rather people keep their hands to themselves, and you won't let anyone touch your ass. If you're ready for a fight, you might even slap someone for it!"
+	gain_text = "<span class='notice'>You'd like it if people kept their hands off your ass.</span>"
+	lose_text = "<span class='notice'>You're less concerned about people touching your ass.</span>"
+	medical_record_text = "Patient demonstrates negative reactions to their posterior being touched."
+	value = 0
+	mob_trait = TRAIT_PERSONALSPACE
+
```

Comment:
If you actually remove Iron Ass, just add this here:
```suggestion
	mob_trait = TRAIT_PERSONALSPACE
	icon = "hand-paper"

```

If you don't, you need to find another icon for this one.